internal/internal_workflow_testsuite.go

```diff
@@ -1044,6 +1044,12 @@ func (env *testWorkflowEnvironmentImpl) RequestCancelWorkflow(domainName, workfl
 	return nil
 }
 
+func (env *testWorkflowEnvironmentImpl) SignalExternalWorkflow(domainName, workflowID, runID, signalName string, input []byte, callback resultHandler) {
+	// fake testing to mimic signal external workflow.
+	env.signalHandler(signalName, input)
```

Comment:
who does this work?

Reply:
Removed fake test from this pull request, will do the test on issue: https://github.com/uber-go/cadence-client/issues/347

Reply:
I added sample for "integration test" of this, also tried disabling the sticky execution, results are good